Sources/FluentSQL/FluentSQLQuery.swift

```diff
@@ -16,7 +16,7 @@ public protocol FluentSQLQuery {
     var groupBy: [GroupBy] { get set }
     var limit: Int? { get set }
     var offset: Int? { get set }
-    var values: [String: Expression] { get set }
+    var values: [[String: Expression]] { get set }
```

Comment:
This is a public protocol. So we unfortunately can't change it unless we do a major version bump.